shared/settings/manage-contacts.native.tsx

```diff
@@ -39,13 +41,23 @@ const ManageContacts = (_: Props) => {
     [dispatch, contactsImported, status]
   )
   const onOpenAppSettings = React.useCallback(() => dispatch(ConfigGen.createOpenAppSettings()), [dispatch])
+  const onStartChat = React.useCallback(() => {
+    dispatch(RouteTreeGen.createSwitchTab({tab: Tabs.chatTab}))
```

Comment:
I'm not sure whether we want to go to the chat tab here - I added this because it felt weird to have the back button from a created conversation lead to contact settings. Could go either way. cc @cecileboucheron ?

Reply:
Talked IRL, this is fine.